packages/react/etc/react.api.md

```diff
@@ -841,6 +841,8 @@ export class DetailsListBase extends React_2.Component<IDetailsListProps, IDetai
     render(): JSX.Element;
     // (undocumented)
     scrollToIndex(index: number, measureItem?: (itemIndex: number) => number, scrollToMode?: ScrollToMode): void;
+    // (undocumented)
+    updateColumn(column: IColumn, width?: number, newColumnIndex?: number): void;
```

Comment:
I know the other methods aren't done like this, but it would be more future-compatible to take the `width` and `newColumnIndex` in a `newProps` object:
```ts
updateColumn(column: IColumn, newProps: {
  width?: number;
  newColumnIndex?: number;
});
```
That way a caller doesn't have to slot `undefined` into a position, and we can add more update-able fields going forward.